### ISSUE TITLE
ARIA-disabled is not inclusive language

### ISSUE THREAD

**Issue by sheribyrne** [w3c/aria#1435]
ARIA-disabled is used when something isn't supposed to be accessed.  It's not the most inclusive language, because it implies disabled==doesn't work.  Would like to see ARIA-inactive or ARIA-unavailable instead

**Comment by schne324** [comment]
I understand that the important factor here is everyone's _perception_ of the meaning of the word "disabled" (in the context of the `aria-disabled` attribute) but I assume the initial intent aligned with the below definition:

> **Disabled**: _of a device or mechanism:_ rendered inoperative
> \- Merriam-Webster

side note: webster has a specific technical definition of the word "enabled":

> _technical_ : to cause (a feature or capability of a computer) to be active or available for use

**Comment by cookiecrook** [comment]
Trying to capture discussion from the ARIA call today... 

Opinions on whether the change _should_ be made were mixed. 

Some areas that I think the group has general agreement:
1. `aria-enabled` would be a simple enough change, but we could only deprecate, never remove `aria-disabled` for compatibility reasons.
2. The connection to `html:input[disabled]` would be much more difficult to change, and therefore it's interplay with `aria-enabled` could cause author misuse, leading to real-world harm in the form of end-user failure, and author cognitive load.
3. If changed, it should be changed in HTML and other W3C specs as part of the same larger effort. 
4. If changed, there are other terms (`aria-invalid`, etc) that should be considered at the same time.
